packages/material-ui-unstyled/src/composeClasses/composeClasses.js

```diff
@@ -0,0 +1,21 @@
+function composeClasses({ slots = {}, classes = {}, getUtilityClass = () => '' }) {
```

Comment:
Do we need these defaults?

```suggestion
function composeClasses({ slots, classes, getUtilityClass }) {
```

Reply:
Actually we need only for the classes, not for the other.

Reply:
```suggestion
function composeClasses({ slots, classes = {}, getUtilityClass }) {
```

Reply:
So should we simply this line, we don't need to test classes (or the opposite)?

`          if (classes && classes[key]) {`

Reply:
Sorry, removed the defaults, you were correct 👍